runtime/lua/vim/treesitter.lua

```diff
@@ -108,6 +108,20 @@ function M.create_parser(bufnr, lang, id)
   return self
 end
 
+--- Gets the parser for this bufnr / ft combination.
+--
+-- If needed this will create the parser.
+-- In any case, the provided callbacks will be attached to the buffer.
```

Comment:
what if there are already callbacks ? `unconditionnally attach the provided callback`

Reply:
Then the provided callbakc will be called _after_ the already attached callback.